week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -40,8 +40,26 @@
 function insertPeopleData(arrayOfPeople) {
   let content = document.querySelector("#content");
   //Write your code in here
+  //console.log(content);
+  
+  
+  for(var i = 0; i < arrayOfPeople.length; i++){
+    //console.log(arrayOfPeople.length);
+
+    let nameEls = document.createElement("h1");
```

Comment:
```suggestion
    const nameEls = document.createElement("h1");
```
Consider using `const` here as the variable may not need to be re-assigned.